src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
@evsh I don't mind implementing this, as I do prefer doing things "the right way". That said I don't think doing something like `boost::program_options` like you describe will work here, as I need an unspecified option to have a value that explicitly means it was undefined rather than an implicit true / false value. So think it would be better to utilize the existing `TriStateBool` class (which exists for exactly that purpose) and create a new kind of option based on that.